documentation/docs/configuration.md

```diff
@@ -62,6 +62,84 @@ steps:
     newmanGlobals: 'myNewmanGlobals'
 ```
 
+## Sending log data to the SAP Alert Notification Service (ANS) on BTP
+
+The SAP Alert Notification Service (ANS) on BTP allows a user to define
+certain actions, such as for example sending email or triggering http
+requests, on incoming events. If the ANS service key is configured in Piper,
+any Piper-go step will send log data to the ANS backend.
+
+The following translation from log message to ANS event is performed:
+
+- `eventType`: defaults to 'Piper'; can be overwritten with the event template
+- `eventTimestamp`: the time of the log entry
+- `severity` and `category`: here the log level is translated:
+
+  | log level | severity | category |
+  |-----------|----------|----------|
+  | info | INFO | NOTICE |
+  | debug | INFO | NOTICE |
+  | warn | WARNING | ALERT |
+  | error | ERROR | EXCEPTION |
+  | fatal | FATAL | EXCEPTION |
+  | panic | FATAL | EXCEPTION |
+
+- `subject`: defaults to the step name; can be overwritten with the event template
+- `body`: the log message
+- `priority`: not set by Piper, can be set with the event template
+- `region`: cannot be set
+- `regionType`: cannot be set
+- `tags`: optional key value pairs. The following are set by Piper:
+  - `logLevel`: the logrus log level
+  - `ans:correlationId`: set to the Piper correlation ID
+  - `ans:sourceEventId`: set to the Piper correlation ID
```

Comment:
Just so I understand, what is the correlation ID? 😅 Why do we need it?

Reply:
I will add some text. Usually it is the build url. It's supposed to be a fairly unique identifier of the pipeline run.